src/Microsoft.Performance.SDK.Runtime/Extensibility/SourceDataCookerDependencyProvider.cs

```diff
@@ -33,9 +33,22 @@ public TOut QueryOutput<TOut>(DataOutputPath dataOutputPath)
             return (TOut)QueryOutput(dataOutputPath);
         }
 
+        public bool TryQueryOutput<TOut>(DataOutputPath dataOutputPath, out TOut result)
```

Comment:
We seem to be using just 'T' on the other methods instead of 'TOut.' Should we use just 'T' here for consistency?

Reply:
This class is already using T as a type parameter, so for these methods, we use TOut instead.